src/js/components/Card.js

```diff
@@ -150,10 +216,12 @@ export default class Card extends Component {
 
 Card.propTypes = {
   thumbnail: PropTypes.string,
-  description: PropTypes.string,
-  heading: PropTypes.string,
+  textSize: PropTypes.oneOf(['small', 'medium', 'large', 'xlarge']),
   label: PropTypes.string,
-  link: PropTypes.element,
+  heading: PropTypes.string,
+  headingStrong: PropTypes.bool,
+  paragraph: PropTypes.string,
```

Comment:
Given that paragraph and description are both rendered as Paragraph, I think we should make these properties more semantically named. Perhaps "summary" and "details"? I'm open to suggestions.